### ISSUE TITLE
Please add a clipboard timer 

### ISSUE THREAD

**Issue by prorounak2005** [rickyrockrat/parcellite#78]
Parcellite is awesome. It would mean a lot to me if the clipboard timer could be added. For example whenever a user copies something and the timer is enabled for 10 seconds. The copied text will remain till 10 seconds and it will get automatically removed after 10 secs. It is very important because sometimes we copy passwords and it is important to remove those passwords from our clipboard manager.

**Comment by Massimo-B** [comment]
Yes, because of https://github.com/keepassxreboot/keepassxc/issues/9842 this might be useful.
